.github/workflows/codecov.yml

```
@@ -26,44 +26,24 @@ jobs:
-            patchelf \
-            lcov
-      - name: Install cargo-llvm-cov
-        uses: taiki-e/install-action@cargo-llvm-cov
-      - name: Ensure coverage directory exists
-        run: mkdir -p coverage
-      - name: Generate Rust coverage (lib tests)
-        working-directory: src-tauri
-        run: cargo llvm-cov --lib --lcov --output-path ../coverage/rust-lib-lcov.info
-      - name: Generate Rust coverage (doc tests)
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.8)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)